tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayWorkspaceProjectProvider.java

```diff
@@ -113,7 +115,7 @@ public IStatus importProject( String location, IProgressMonitor monitor, String
 
                 IProject project = CoreUtil.getProject( path.lastSegment() );
 
-                if( !bundleUrl.equals( defaultBundleUrl ) )
+                if( bundleUrl.equals( defaultBundleUrl ) )
```

Comment:
I'm not sure I understand this logic.  Can you please add a junit test (not a UI test) but a headless unit test for this one?  Thanks  see the last commits on this branch here.  

https://github.com/gamerson/liferay-ide/tree/mavenModule

Especially this commit which contains a new unit test where I am importing a new liferay workspace project.  For this unit test for this PR we need to create a new workspace with custom URL and one with default and verify that the correct version of portal is configured in initBundle